pymess/config.py

```diff
@@ -72,6 +72,7 @@
     'EMAIL_PULL_INFO_DELAY_SECONDS': 60 * 60,  # 1 hour
     'EMAIL_PULL_INFO_MAX_TIMEOUT_FROM_SENT_SECONDS': 60 * 60 * 24 * 30,  # 30 days
     'EMAIL_RETRY_SENDING': True,
+    'EMAIL_STORAGE_PATH' : 'pymess/emails',
```

Comment:
Not sure if to keep this as a config value. Can be set only before initial migration, any changes later will generate new migration and possibly a need to move existing files manually. But I guess it's better to have it hardcoded on two places.